sdk-wallet/validator_signer.md

```diff
@@ -0,0 +1,14 @@
+## ValidatorSigner
+
+```
+class ValidatorSigner:
+    constructor(secret_key: ValidatorSecretKey);
+
+    // Named constructors
+    static fromPemFile(path: Path, index: number = 0): ValidatorSigner;
+
+    sign(data: bytes): bytes;
```

Comment:
same as for userSigner

Reply:
Fixed in the separate PR for sdk-wallet, now brought here.